src/renderer/components/ForwardToList.js

```diff
@@ -0,0 +1,47 @@
+const React = require('react')
+const { ipcRenderer } = require('electron')
+const SearchableList = require('./SearchableList')
+
+const ChatListItem = require('./ChatListItem')
+
+class ForwardToList extends SearchableList {
+  constructor (props) {
+    super(props)
+    this.state.showVerifiedContacts = false
+    this.handleSearch = this.handleSearch.bind(this)
+    this.search = this.search.bind(this)
+  }
+
+  _getData () {
+    return ipcRenderer.sendSync(
```

Comment:
sendSync is bad, but maybe okay for the moment. I would like to see the Store pattern here that @nicodh introduced.

Reply:
Agreed. Lets try to use this pattern whenever we implement new things.

Reply:
I use the same thing as in the contacts list... Also this isn't using the same data as the chatlist, because it's a different list.
so I see no real advantage in using a store for this. Can you explain why you wanna use it here?

Reply:
ralphtheninja already said it. We should move to the store pattern step by step if we refactor or implement new stuff. One advantage is that it's easier to maintain a software which has a homogeneous architecture than one using different pattern at different parts...

Reply:
@Simon-Laux if you want we can look into it together tomorrow in brombi if you have time :)

Reply:
I will provide a suggestion how to implement contactList and forwardChatList with a store. Then you can review it together, maybe...